vncdotool/api.py

```diff
@@ -82,32 +81,38 @@ def disconnector(protocol: VNCDoToolClient) -> None:
     def __getattr__(self, attr: str) -> Any:
         method = getattr(self.factory.protocol, attr)
 
-        def errback(reason: Failure, *args: Any, **kwargs: Any) -> None:
-            self.queue.put(Failure(reason))
-
-        def callback(protocol: VNCDoToolClient, *args: Any, **kwargs: Any) -> Deferred:
+        def threaded_call(protocol: VNCDoToolClient, *args: Any, **kwargs: Any) -> Deferred:
             def result_callback(result: V) -> V:
                 self.queue.put(result)
                 return result
             d = maybeDeferred(method, protocol, *args, **kwargs)
             d.addBoth(result_callback)
             return d
 
-        def proxy_call(*args: Any, **kwargs: Any) -> Any:
-            reactor.callFromThread(self.factory.deferred.addCallbacks,
-                                   callback, errback, args, kwargs)
+        def errback_not_connected(reason: Failure, *args: Any, **kwargs: Any) -> Failure:
+            self.queue.put(reason)
+            return reason
+
+        def callable_threaded_proxy(*args: Any, **kwargs: Any) -> Any:
+        reactor.callFromThread(
+            self.factory.deferred.addCallbacks,  # ensure we're connected
+            threaded_call,
+            errback_not_connected,
+            args,
+            kwargs,
+        )
```

Comment:
The indention is again wrong here. 
```suggestion
        def callable_threaded_proxy(*args: Any, **kwargs: Any) -> Any:
            reactor.callFromThread(
                self.factory.deferred.addCallbacks,  # ensure we're connected
                threaded_call,
                errback_not_connected,
                args,
                kwargs,
            )
```